lib/config/config_mgmt.go

```diff
@@ -0,0 +1,101 @@
+package config
```

Comment:
The base functionality for managing in-memory config data that is backed up to a file.

Reply:
I would love a different word for this because 'config' is already overloaded. I would like to do the new name after this PR merges.